### DISCUSSION TITLE
Bootstrap 5 beta2 is out and about updating

### DISCUSSION THREAD

**Discussion by 1337180** [joomla/joomla-cms#32385] [General] [answered]
hey,
beta 2 is out - maybe final beta, lots of testing was done
bs5.1 is already on the target list
how will j4 keep on updates with BS?
I mean will we have to wait until 4.1 to update for example on bs5.2? Or will this be in the minor 4.0.x release cycles.
I mean there can be also some system similar to definitions on AV/Amalware like independent depencencies/packages update list.
Worth of think about this. Why not. Its better.

**Comment by richard67** [comment, marked as answer]
https://github.com/joomla/joomla-cms/pull/32378

**Reply by richard67** [reply]
As you can see when following the above link, the update to BS 5 Beta 2 has already happened, and if you would check how we handle dependencies in Joomla 3 and 4, you would see that we already do since long time what you are talking about. We regularly update our dependencies with respect to semver and backwards compatibility, so if BS 5 Beta 3 will be out, we will for sure update our dependency to that and so on. We only wont update later to a BS 5.1, 5.2, ... or 6, if that would have braking changes compared to 5.0.x (beta or rc or whatever).

I would expect people to research first if their requests are valid before keeping CMS maintainer busy with posting them.

**Comment by ghost** [comment]
please mark it as answered

**Reply by richard67** [reply]
It needs to be moved to another category first. Ideas cannot be marked as answered.
